database/gormdb_test.go

```diff
@@ -478,35 +479,37 @@ func TestGormDBAssociateUserWithRemoteIdentity(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !reflect.DeepEqual(&user1, wantUser1) {
-		t.Errorf("have user %+v want %+v", &user1, wantUser1)
+	if diff := cmp.Diff(wantUser1, gotUser1, protocmp.Transform()); diff != "" {
+		t.Errorf("CreateUserFromRemoteIdentity() mismatch (-wantUser, +gotUser):n%s", diff)
```

Comment:
Change wantUser to wantUser1 and gotUser to gotUser1 ?

Reply:
Isn't it supposed to be like this? Given that wantUser1 and gotUser1 are simply the variable names used for the test.

Reply:
Not sure if there is a misunderstanding here or not, but I like to keep the string text and the variable names the same. Although it is bit annoying to keep them in sync when you rename variables…

Reply:
Ok. Skal være fikset nå